packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.tsx

```diff
@@ -72,10 +76,30 @@ export class GuxTooltip {
     trackComponent(this.root, { variant: this.placement });
   }
 
+  componentDidLoad(): void {
+    if (!this.tooltipObserver && this.baseTooltip) {
```

Comment:
Will there ever be a time when `this.tooltipObserver` is true when this code block runs? Trying to think if on compnoentDidLoad if `this.tooltipObserver` is always going to be false since `this.tooltipObserver.disconnect();` is called on line 99. Maybe I'm wrong though but just wanted to check.

Reply:
I think that makes sense. I've updated it again